document-generator-core/swagger-2.0/spec/document-generator.json

```diff
@@ -0,0 +1,57 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Document generator API",
+    "description": "An API for generation of documents",
+    "version": "1.0.0"
+  },
+  "host": "localhost",
+  "schemes": [
+    "http"
+  ],
+  "tags": [
+    {
+    "name": "document-generator"
+    }
+  ],
+  "paths": {
+    "/document-generate": {
+      "post": {
+        "x-operationName": "generate",
+        "summary": "Generation of documents",
+        "description": "Generate a document using the request to obtain the document data and send to render service",
```

Comment:
`Takes a URI of a resource and synchronously generates a document of the requested mime type`